src/components/UserInfo.js

```diff
@@ -1,21 +1,24 @@
 export default class UserInfo {
-  constructor(nameSelector, jobSelector) {
+  constructor(nameSelector, jobSelector, avatorselector) {
     this._name = document.querySelector(nameSelector);
     this._job = document.querySelector(jobSelector);
+    this._avatar = document.querySelector(avatorselector);
   }
 
   //   returns an object with information about the user to display the user data in the open form.
   getUserInfo() {
     this._userInfo = {
       name: this._name.textContent,
-      description: this._job.textContent,
+      about: this._job.textContent,
+      avatar: this._avatar.src,
```

Comment:
Needs correction: You should create a separate getter for the avatar `getAvatar`, since it's controlled by the different form

Reply:
Got it.